www/src/templates/template-api-markdown.js

```diff
@@ -0,0 +1,251 @@
+      .filter(def => def.kind !== `typedef` && def.memberof)
+      .map(def => {
+        if (!context.apicalls) {
+          // When an api call list is not available, the line numbers from jsdoc
+          // might be useful. Just for actions.mdx right now.
+          def.codeLocation.file = node.relativePath
+          if (!def.codeLocation.file) {
+            def.codeLocation = null
+          }
+        } else {
+          // API pages having apicalls did not query for this in the page query,
+          // so just remove it instead. Having one that returns nothing supresses
+          // documentation git links completely.
+          def.codeLocation = null
+        }
+        return def
+      })
+    return acc.concat(doc)
+  }, [])
+
+  let funcs = sortBy(docs, func => func.name)
+
+  const mergedFuncs = funcs.map(func => {
+    return {
+      ...func,
+      ...normalized.find(n => n.name === func.name),
+    }
+  })
+
+  return mergedFuncs
+}
+
+function DocsTemplate({ data, location, pageContext }) {
+  const { next, prev } = pageContext
+  const page = data.mdx
+  const [urlSegment, itemList] = getDocsData(location)
+  const toc =
+    !page.frontmatter.disableTableOfContents && page.tableOfContents.items
+
+  // Cleanup graphql data for usage with API rendering components
+
+  return (
+    <React.Fragment>
+      <Helmet>
+        <title>{page.frontmatter.title}</title>
+        <meta name="description" content={page.excerpt} />
+        <meta property="og:description" content={page.excerpt} />
+        <meta property="og:title" content={page.frontmatter.title} />
+        <meta property="og:type" content="article" />
+        <meta name="twitter:description" content={page.excerpt} />
+        <meta name="twitter.label1" content="Reading time" />
+        <meta name="twitter:data1" content={`${page.timeToRead} min read`} />
+      </Helmet>
+      <Layout
+        location={location}
+        itemList={itemList}
+        enableScrollSync={urlSegment === `docs` ? false : true}
+      >
+        <DocSearchContent>
+          <Container
+            overrideCSS={{
+              pb: 0,
+              [mediaQueries.lg]: {
+                pt: 9,
+              },
+              [toc && mediaQueries.xl]: {
+                ...containerStyles,
+              },
+            }}
+          >
+            <Breadcrumb location={location} itemList={itemList} />
+            <h1 id={page.fields.anchor} sx={{ mt: 0 }}>
+              {page.frontmatter.title}
+            </h1>
+          </Container>
+          <Container
+            overrideCSS={{
+              pt: 0,
+              position: `static`,
+              [mediaQueries.lg]: {
+                pb: 9,
+              },
+              [toc && mediaQueries.xl]: {
+                ...containerStyles,
+                display: `flex`,
+                alignItems: `flex-start`,
+              },
+            }}
+          >
+            {toc && (
```

Comment:
I'd rather keep it simple and get rid of all the parts that aren't necessary for the API pages, like the table of contents and the "stub page"